vita: Add new button map using touchscreen

d3m3vilurr opened the pull request:
- L2(LT): Back half left-top
- R2(RT): Back half right-top
- L3(LS CLK): Back half left-botton
- R3(RS CLK): Front half right-bottom

d3m3vilurr committed:
vita: Add new button map using touchscreen

- L3(LS CLK): Front half left
- R3(RS CLK): Front half right
- L2(LT): Back half left
- R2(RT): Back half right

xyzz commented:
Is it compatible with various vita attachments? Like http://images.pushsquare.com/news/2016/04/this_slick_vita_grip_adds_triggers_for_better_ps4_remote_play/large.jpg

d3m3vilurr commented:
hm...that is not compatible.. cuz, i never used remote play. so i don't know real map..
but i can patch this. :)

d3m3vilurr committed:
vita: All touch map move to back touch screen

http://images.pushsquare.com/news/2016/04/this_slick_vita_grip_adds_triggers_for_better_ps4_remote_play/large.jpg

d3m3vilurr commented:
@xyzz: hmm... i patched.
but https://youtu.be/iiH1BKmt9gk?t=2m38s this movie not matched that picture.
I will try to find actual virtual button layout...

xyzz commented:
It's probably configurable in settings. But basically if it's top=l2/r2, bottom=l3/r3 I guess it's fine.

d3m3vilurr commented:
ok. i don't know each button adjust point. so i just split half and half. :)
but i think we need configure menu about this.

also, i think, vita remote play feature have multiple type of button maps, many screenshot shown about this.
for example [witcher3 only use backscreen](https://thegamerwithkids.com/2015/05/19/the-witcher-3-wild-hunt-not-as-good-with-remote-play-on-the-ps-vita/) but [gta v use frontscreen](http://goingsony.com/stories/10554-gta-v-remote-play-controls) for l3/r3
